pl_examples/README.md

```diff
@@ -51,3 +51,6 @@ Contains implementations leveraging [loop customization](https://pytorch-lightni
 
 - [K-fold Cross Validation Loop](./loop_examples/kfold.py): Implemenation of cross validation in a loop and special datamodule.
 - [Yield Loop](./loop_examples/yielding_training_step.py): Enables yielding from the training_step like in a Python generator. Useful for automatic optimization with multiple optimizers.
+## quantization
```

Comment:
Can we call this something like "Intel quantization" instead of "quantization" everywhere, to reduce confusion between this workflow and PyTorch's native quantization workflow?

Reply:
Done